src/MagentoDataLayerUnsubscribeManager.ts

```diff
@@ -0,0 +1,18 @@
+import { MagentoDataLayerBase } from "./MagentoDataLayerBase";
+
+import { ADD_TO_CART } from "./eventNames";
```

Comment:
or similarly here `actions.ADD_TO_CART` but I also don't know how I feel about making it even harder to access (object.key vs just the var) 

Have you considered this, what do you think ?